packages/services/api/src/modules/integrations/index.ts

```diff
@@ -10,5 +9,5 @@ export const integrationsModule = createModule({
   dirname: __dirname,
   typeDefs,
   resolvers,
-  providers: [SlackIntegrationManager, GitHubIntegrationManager, AuditLogManager],
+  providers: [SlackIntegrationManager, GitHubIntegrationManager],
```

Comment:
No need to to add `AuditLogManager`, since we removed the modules dependency upon it.